### ISSUE TITLE
Advanced Editorial statistics for OPS

### ISSUE THREAD

**Issue by ajnyga** [pkp/pkp-lib#6295]
The default OJS Editorial statistics were cleaned out with https://github.com/pkp/pkp-lib/issues/6171. 

This ticket is for tracking any future development on OPS Editorial statistics page. 

The current default layout now shows figures for:
Submissions Received 
Submissions Accepted
Submissions Declined
Submissions Published

@alexxxmendonca I think you had some ideas about this? Maybe @NateWr too?

**Comment by NateWr** [comment]
Filtering by category may be important for OPS.

**Comment by alexxxmendonca** [comment]
> Filtering by category may be important for OPS.

Yes. Also:

Country/Region
Language
Whether it was rejected with or without moderation
Submitted by author vs journal (this can be tricky as we are not collecting this data)

Here are some of the data important for us:

![image](https://user-images.githubusercontent.com/4059679/96718031-cb71ba80-137d-11eb-8363-0c160a2434f6.png)
![image](https://user-images.githubusercontent.com/4059679/96718041-cf9dd800-137d-11eb-8e05-b15cb4aa27e4.png)
Languages and acceptance ratio based on languages

![image](https://user-images.githubusercontent.com/4059679/96718085-da586d00-137d-11eb-88c6-b94496e95700.png)
Geographical distribution and acceptance ratio based on that

![image](https://user-images.githubusercontent.com/4059679/96718137-f2c88780-137d-11eb-97f4-c42077949e55.png)
Same for disciplines (sections and categories)

Also, please refrain from using "published".

**Comment by ajnyga** [comment]
> Whether it was rejected with or without moderation
This is probably difficult. OPS only knows a distinction between an unpublished and published submission and there is no status for a submission that has passed moderation.

> Also, please refrain from using "published".
Yes, I would like to change the backend language all at once so I recentely requested Alec to look at this issue and let me know how to proceed: https://github.com/pkp/pkp-lib/issues/5610

**Comment by alexxxmendonca** [comment]
> This is probably difficult. OPS only knows a distinction between an unpublished and published submission and there is no status for a submission that has passed moderation.

Yes, I understand. Perhaps there could be something to distinguish submissions that are rejected "at first" from those that are sent to moderation?

This is similar to the concept of "desk reject" vs "reject after peer review" in a journal.

**Comment by ajnyga** [comment]
The thing is that by default all submissions are in a way moderated if no screening plugins are applied, only the moderator can publish. OPS does not record things like "changes requested" or "revised version uploaded" at the moment. This is also connected to https://github.com/pkp/pkp-lib/issues/5975. 

@asmecher I think we need a general plan on what kind of moderation process OPS core should support and how it would be reflected in Submission status and UI actions etc.

**Comment by alexxxmendonca** [comment]
We've upgraded to OPS 3.3.0.6 and the basic statistics still need revision:

![image](https://user-images.githubusercontent.com/4059679/120536372-6cb3fb00-c3ba-11eb-9c2d-ff7f7782787f.png)

- It says 0 submissions accepted even though we currently have over 900 preprints posted
- It says 623 submissions "published" (correct wording should be "posted") even though we currently have over 900 preprints posted

**Comment by asmecher** [comment]
For usable submission language metadata, we will likely need to resolve https://github.com/pkp/pkp-lib/issues/5000 first.

For geographic distribution, how would we ascertain the submission's country? For example, would this be taken from the submitting author's profile? What about submissions from several assigned (submitting) authors?

For discipline data, there are two requirements:
- https://github.com/pkp/pkp-lib/issues/1550
- Identification of a vocabulary to use. This would need to have language-independent identifiers (e.g. URIs) for each term, sufficient translation coverage (in e.g. an XML form), and a permissive license.

**Comment by alexxxmendonca** [comment]
> For geographic distribution, how would we ascertain the submission's country? For example, would this be taken from the submitting author's profile? What about submissions from several assigned (submitting) authors?

We could start from showing the submitting authors' country. That is what ScholarOne does (there are means to obtain that report via Cognos, though, if you build a separate custom report).

Ideally, however, would be to consider all authors' countries. If this is something that can't be done right now or too complex, we could start with considering only submitting author and being transparent for users running the report that the report takes into consideration only the submitting author country.
